pyccel/ast/numpy_wrapper.py

```diff
@@ -84,25 +86,19 @@ def get_numpy_max_acceptable_version_file():
                 body      = [],
                 results   = [Variable(name = 'b', dtype = NativeBool())])
 
-# https://numpy.org/devdocs/reference/c-api/array.html#c.PyArray_DIMS
-numpy_get_ndims = FunctionDef(name      = 'PyArray_NDIM',
+# Return the shape in the n dimension : function definition in pyccel/stdlib/cwrapper.c
+array_get_dim  = FunctionDef(name    = 'nd_ndim',
                            body      = [],
-                           arguments = [Variable(dtype=PyccelPyArrayObject(), name = 'o', is_pointer=True)],
-                           results   = [Variable(dtype=NativeInteger(), name = 'i')])
-
-# https://numpy.org/devdocs/reference/c-api/array.html#c.PyArray_DATA
-numpy_get_data  = FunctionDef(name      = 'PyArray_DATA',
-                           body      = [],
-                           arguments = [Variable(dtype=PyccelPyArrayObject(), name = 'o', is_pointer=True)],
-                           results   = [Variable(dtype=NativeGeneric(), name = 'v', rank=1)])
-
-# https://numpy.org/devdocs/reference/c-api/array.html#c.PyArray_DIM
-numpy_get_dim   = FunctionDef(name      = 'PyArray_DIM',
-                           body      = [],
-                           arguments = [Variable(dtype=PyccelPyArrayObject(), name = 'o', is_pointer=True),
+                           arguments = [Variable(dtype=NativeVoid(), name = 'o', is_pointer=True),
                                         Variable(dtype=NativeInteger(), name = 'idx')],
                            results   = [Variable(dtype=NativeInteger(), name = 'd')])
 
+# Return the data of ndarray : function definition in pyccel/stdlib/cwrapper.c
```

Comment:
same here, the function definition is located at `pyccel/stdlib/cwrapper/cwrapper_ndarrays.c`.

Reply:
Fixed de6219e